src/main/java/ch/njol/skript/SkriptEventHandler.java

```diff
@@ -255,9 +259,20 @@ static void registerBukkitEvents() {
 			if (e.equals(PlayerInteractAtEntityEvent.class) || e.equals(PlayerArmorStandManipulateEvent.class)) {
 				continue; // Ignore, registered above
 			}
-			if (!registeredEvents.contains(e)) { // Check if event is registered
+
+			HandlerList handlerList;
+
+			try {
+				Method getHandlerListMethod = e.getMethod("getHandlerList");
```

Comment:
The getHandlerList can be private, in which case getMethod doesn't pick up on the method. I recommend looking at `SimplePluginManager.getEventListeners` for how to do this (because simply calling getDeclaredMethod doesn't take subclasses into account.